### ISSUE TITLE
Unable to load the video file

### ISSUE THREAD

**Issue by daspk04** [sagost/VideoUavTracker#1]
Hello,
I have installed this plugin and tried to run using the sample data provided in youtube channel but an error occurred where the video file was not loaded. Snapshot of the error has been attached below. Will be glad if you could help. 
![2017-07-19-002205_929x326_scrot](https://user-images.githubusercontent.com/28738918/28342479-df9fcd90-6c18-11e7-84d5-4d10b7b79e77.png)

**Comment by sagost** [comment]
Hi, do you downloaded ffmpeg binaries and copied them in .qgis3\python\plugins\videouavtracker\FFmpeg. ?

**Comment by daspk04** [comment]
Yes I did download the ffmpeg binaries and copied them into the folder as above. The problem was solved when I downloaded qt5multimedia packages. But still it shows a black screen and video doesn't stream. 
![2017-07-27-175300_1360x728_scrot](https://user-images.githubusercontent.com/28738918/28679843-bf9158f6-72f4-11e7-8a00-0de811341227.png)

It shows my gstreamer installation is missing a plugin & video and audio decoder is not available so I am not sure which plugin should I download so it can stream. My OS is Ubuntu 16.04.2 LTS.
